expr/funcs.go

```diff
@@ -54,7 +57,7 @@ func init() {
 		"aliasByTags":           {NewAliasByNode, true},
 		"aliasByNode":           {NewAliasByNode, true},
 		"aliasSub":              {NewAliasSub, true},
-		"asPercent":             {NewAsPercent, true},
+		"asPercent":             {NewAsPercent, false}, // disabled because it needs an update for runtime normalization and possibly MDP/pre-normalization
```

Comment:
Hmm, this could break some things we have querying our cluster. I think that we will need this re-enabled before we can rollout

Reply:
oh. I believe this can be true again.
asPercent was the toughest one, but I believe I did all the tweaks necessary to make it work in the new model.  But please have a look at it. In the process I also cleaned up the code a bit, for which I created a separate PR, #1616. Workflow wise, let's get that one reviewed/merged first, that will make the diff here smaller.